specification/src/chapter4.adoc

```diff
@@ -2,69 +2,897 @@
+
+| CAT_NNN
+| Resources MAY be assigned to both Normal and Confidential domains (sharing by consent).
+
+| CAT_NNN
+| Resources assigned to a single confidential workload MUST NOT be accessible by any other confidential workload
+
+| CAT_NNN
+| Resources MAY be assigned to multiple confidential workloads (sharing by consent)
+
+|===
+
+RISC-V hardware enforced isolation mechanisms can be used as follows to meet those guarantees:
+
+[width=100%]
+[%header, cols="5,20"]
+|===
+| ID#     
+| Requirement
+
+| CAT_NNN 
+| PMP/ePMP or MTT MUST be used to isolate Root domain from other domains.
+
+| CAT_NNN  
+| Supervisor domains MUST be used to enforce isolation between Normal and Confidential domains.
+
+|===
+
+See xref:chapter3.adoc#_supervisor_domains[supervisor domains].
+
+NOTE: MTT can be sufficient for protecting Root domain in the sense that M-mode can enforce that its own resources are never assigned to another domain. PMP/ePMP still add further protections for M-mode, such as the ability to implement temporal isolation boundaries within M-mode (for example, protect early boot code), or to prevent itself from accessing or executing from memory assigned to lower privilege levels (privilege escalation).
+
+[width=100%]
+[%header, cols="5,20"]
+|===
+| ID#     
+| Requirement
+
+| CAT_NNN
+| Hypervisor extension MUST be supported
+
+| CAT_NNN
+| MMU MUST be used to enforce isolation between Confidential guests within Confidential domain.
+|===
+
+==== Root of trust
+
+See xref:chapter2.adoc#_reference_model[reference model].
+
+[width=100%]
+[%header, cols="5,20"]
+|===
+| ID#     
+| Requirement
+
+| CAT_NNN 
+| A CoVE system MUST implement a HW RoT
+
+|===
+
+==== Authorized Boot
+
+See xref:chapter2.adoc#_authorized_software[authorized software].
+
+[width=100%]
+[%header, cols="5,20"]
+|===
+| ID#     
+| Requirement
+
+| CAT_NNN 
+a| Confidential guests MUST not boot until at least the security platform has been verified:
+
+* TSM in Confidential domain
+* Root domain
+* Boot state of all trusted subsystems
+|===
+
+Boot in a cloud hosting context is typically based on:
 
+* Measured boot of a hosting platform, including Root domain and TSM
+* Platform attestation and security provisioning (unsealing) by a remote provisioning system
+* Launch and measurement of confidential workloads, only once the system has been unsealed
+
+A _trusted platform module_ (TPM) can be used to measure the security platform.
+
+Measuring confidential guests can be done by TSM in Confidential domain.
+
+The process can involve multiple stages (layered boot). 
+
+==== Attestation
+
+See xref:chapter2.adoc#_attestable_services[attestable services].
+
+Attestation of confidential workloads is typically layered, for performance and separation of concern:
+
+* A security platform attestation, signed by a hardware root of trust
+* A confidential workload attestation, signed by TSM
```

Comment:
```suggestion
* A confidential workload attestation, signed by TSM, where the TSM is authenticated by the HW RoT
```

Reply:
This would duplicate the bullet above. "Security platform" is defined in the next paragraph.

Reply:
then we should move that one above this one.

Reply:
That breaks the flow. The two bullets introduce the concept of layering. Then the individual components are expanded and defined in more detail. The same format is used throughout the document. Changing the format here means changing throughout the document for consistency. I would rather not do that unless there is some major issue here. I don't see one?